decidim/decidim
# Do not send upcoming meeting events notification for past events

#### :tophat: What? Why?
When creating a past Meeting, the user that follows the space receives an upcoming meeting notification that is no longer required. This PR fixes this. 

#### Testing
Scenario 1:
- Go to the public create meeting form and add a past event. 
- The space followers should not receive any upcoming notification  

Scenario 2:
- Go to the public meetings page
- Edit a meeting and set the start date in the past
- The space followers should not receive any upcoming notification 

Scenario 3: 
- Go to Admin meetings section 
- Create a past meeting and publish it 
- The space followers should not receive any upcoming notification 

Scenario 4:
- Go to Admin meetings section 
- Edit an event, and set the start date in the past 
- The space followers should not receive any upcoming notification 

:hearts: Thank you!
